src/CMakeLists.txt

```diff
@@ -43,7 +43,7 @@ add_executable(xournalpp exe/Xournalpp.cpp)
 target_link_libraries(xournalpp PRIVATE xoj::core xoj::util)
 set_target_properties(xournalpp PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}"
         MACOSX_BUNDLE $<IF:$<PLATFORM_ID:Darwin>,TRUE,${CMAKE_MACOSX_BUNDLE}>
-        WIN32_EXECUTABLE $<IF:$<PLATFORM_ID:Windows>,TRUE,${CMAKE_WIN32_EXECUTABLE}>
+        WIN32_EXECUTABLE $<IF:$<AND:$<PLATFORM_ID:Windows>,$<NOT:$<CONFIG:Debug>>>,TRUE,${CMAKE_WIN32_EXECUTABLE}>
```

Comment:
Xournalpp should be connecting to the console, which started it, does this not work for you?
It should be connected to the vscode terminal.